src/components/include/application_manager/reset_timeout_handler.h

```diff
@@ -33,8 +33,28 @@
 #ifndef SRC_COMPONENTS_INCLUDE_APPLICATION_MANAGER_RESET_TIMEOUT_HANDLER_H_
 #define SRC_COMPONENTS_INCLUDE_APPLICATION_MANAGER_RESET_TIMEOUT_HANDLER_H_
 
+#include <stdint.h>
+
 namespace application_manager {
 namespace request_controller {
+
+/*
+ * @brief Structure for mobile request
```

Comment:
@LitvinenkoIra too pure description, another developer may be confused . Please specify for what purpose this struct will be used.

Reply:
@LuxoftAKutsan description changed 54ad2b0